frontend/views/containers/chatroom/MessageBase.vue

```diff
@@ -273,6 +297,10 @@ export default ({
     color: var(--text_2);
     border-color: var(--text_1); // var(--text_2);
   }
+
+  & .c-mention {
```

Comment:
it seems`&` here is a typo or safe to be removed. :)

Reply:
Correct. That's not necessary.